task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     tekton.dev/pipelines.minVersion: 0.12.1
     tekton.dev/tags: image-build, konflux
   labels:
-    app.kubernetes.io/version: "0.2"
+    app.kubernetes.io/version: 0.2.1
```

Comment:
We don't track minor versions yet, this can stay at 0.2

Reply:
Reverted

Reply:
How this has been reverted? Still seeing `0.2.1`

Reply:
In the end we decided not to revert. Did we break something?

Reply:
No this part of the code didn't break anything...was just going through the code and find it weird so asked it here